src/platform/plugins/shared/discover/public/context_awareness/profile_providers/observability/observability_root_profile/accessors/get_default_ad_hoc_data_views.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { ObservabilityRootProfileProvider } from '../types';
+
+const ALL_LOGS_DATA_VIEW_ID = 'discover-observability-root-profile-all-logs';
+
+export const getDefaultAdHocDataViews: ObservabilityRootProfileProvider['profile']['getDefaultAdHocDataViews'] =
+
+    (prev, { context }) =>
+    () => {
+      const prevDataViews = prev();
+
+      if (!context.allLogsIndexPattern) {
+        return prevDataViews;
+      }
+
+      return prevDataViews.concat({
+        id: ALL_LOGS_DATA_VIEW_ID,
```

Comment:
Would be great if links from Observability start point to this ad hoc data view instead of creating another one.

<img width="1543" alt="Screenshot 2025-01-23 at 13 26 30" src="https://github.com/user-attachments/assets/7919d3b1-4656-45cc-a5e4-e56092aa0256" />